wetterdienst/file_path_handling/path_handling.py

```diff
@@ -1,20 +1,24 @@
 """ functions to handle paths and file names"""
 from pathlib import Path, PurePosixPath
 from typing import Union, List
+import datetime as dt
+
 from bs4 import BeautifulSoup
+import pandas as pd
 
 from wetterdienst.constants.access_credentials import (
```

Comment:
() are redundant parentheses here. Or are there any benefits ?

Reply:
Why are they redundant? Without them it would be

HTTPS_EXPRESSION, \
DWD_SERVER, \
....

black code formatting uses them for multiple imports from one module in order to create this "import by line" format.